### ISSUE TITLE
Changing game speed on Windows works only once

### ISSUE THREAD

**Issue by bunnybot** [widelands/widelands#1499]
After starting a new game in Windows 7 64 bit, pushing Pg Up has the same effect as keeping it pressed, so the game speed starts racing up.

This can then be stopped by pushing Pg Down, and the game speed can be adjusted.

This can be done only once per game, after that both Pg Up and Pg Down are ignored.

This behaviour was already present in Build17, which is the first build I ever played.

------------------------------------
Imported from Launchpad using lp2gh.

 * date created: 2014-03-08T06:12:47Z
 * owner: gunchleoc
 * the launchpad url was https://bugs.launchpad.net/bugs/1289699

**Comment by bunnybot** [comment]
(by hjd)
I have no problems turning the speed up or down during the course of the game on Ubuntu. Sounds like a Windows-specific issue. Sounds pretty bad if it has been this way for a while though.

**Comment by bunnybot** [comment]
(by gunchleoc)
Yes, it's definitely Windows-related. I have no trouble on Ubuntu.

**Comment by bunnybot** [comment]
(by qcumber-some)
I can not see this behaviour on Linux (with b18 and trunk) and not on Wine (with b18).

Might also be something related to your system (hardware, drivers...)

**Comment by bunnybot** [comment]
(by gunchleoc)
I just tried my other Windows machine (which usually runs Linux :P), and it's working fine there. Let's make this low priority then.

**Comment by bunnybot** [comment]
(by sirver)
Setting to incomplete. If it times out without any other mentions of this problem we'll close as invalid and put it down to something on Guns computer making problems.

**Comment by bunnybot** [comment]
(by janitor)
[Expired for widelands because there has been no activity for 60 days.]
